modules/apps/27-interchain-accounts/controller/ibc_middleware.go

```diff
@@ -75,15 +75,15 @@ func (im IBCMiddleware) OnChanOpenInit(
 }
 
 // OnChanOpenTry implements the IBCMiddleware interface
-func (im IBCMiddleware) OnChanOpenTry(
-	ctx sdk.Context,
-	order channeltypes.Order,
-	connectionHops []string,
-	portID,
-	channelID string,
-	chanCap *capabilitytypes.Capability,
-	counterparty channeltypes.Counterparty,
-	counterpartyVersion string,
+func (IBCMiddleware) OnChanOpenTry(
+	_ sdk.Context,
+	_ channeltypes.Order,
+	_ []string,
+	_,
+	_ string,
+	_ *capabilitytypes.Capability,
+	_ channeltypes.Counterparty,
+	_ string,
```

Comment:
I think we should probably disable this rule that uses `_` for unused parameters, otherwise it's very difficult to know what each argument is.

Reply:
ok, and thanks!